### PR TITLE
Add docs for Magic HDMI cable SRT/RTMPS playback

### PR DESCRIPTION
**Pull request by irvinebroque**
Minimum viable docs for RTMPS and SRT playback.

Main docs are at the bottom of the "Use your own player" page, and link out to examples using `ffplay`.

### CODE REVIEW

File: content/stream/_partials/_prereqs_first_start_live_streaming.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+{{<Aside type="note">}}
+
+Before you can play live video, you must first be [actively streaming to a live input](/stream/stream-live/start-stream-live)
```
Comment (dcpena) [inline content/stream/_partials/_prereqs_first_start_live_streaming.md]:
```suggestion
Before you can play live video, you must first be [actively streaming to a live input](/stream/stream-live/start-stream-live).
```

File: content/stream/examples/srt_playback.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - Playback
+pcx_content_type: configuration
+title: SRT playback
+weight: 9
+layout: example
+---
+
+{{<render file="_prereqs_first_start_live_streaming.md">}}
+
+Copy the SRT _playback_ URL for your live input from the [Stream Dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs) or the [Stream API](/stream/stream-live/start-stream-live/#using-the-api), and paste it into the URL below, replacing `<SRT_PLAYBACK_URL>`:
```
Comment (dcpena) [inline content/stream/examples/srt_playback.md]:
```suggestion
Copy the **SRT Playback URL** for your live input from the [Stream Dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs) or the [Stream API](/stream/stream-live/start-stream-live/#using-the-api), and paste it into the URL below, replacing `<SRT_PLAYBACK_URL>`:
```

File: content/stream/examples/rtmps_playback.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Copy the RTMPS _playback_ key for your live input from the [Stream Dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs) or the [Stream API](/stream/stream-live/start-stream-live/#using-the-api), and paste it into the URL below, replacing `<RTMPS_PLAYBACK_KEY>`:
+
+```bash
+---
+header: RTMPS playback with ffplay
+---
+ffplay -analyzeduration 1 -fflags -nobuffer -probesize 32 -sync ext 'rtmps://live.cloudflare.com:443/live/<RTMPS_PLAYBACK_KEY>'
+```
+
+For more, see [read the docs](/stream/viewing-videos/using-own-player/#play-live-video-in-native-apps-with-less-than-1-second-latency).
```
Comment (dcpena) [inline content/stream/examples/rtmps_playback.md]:
```suggestion
For more, refer to [Play live video in native apps with less than one second latency](/stream/viewing-videos/using-own-player/#play-live-video-in-native-apps-with-less-than-1-second-latency).
```

File: content/stream/examples/srt_playback.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+```bash
+---
+header: SRT playback with ffplay
+---
+ffplay -analyzeduration 1 -fflags -nobuffer -probesize 32 -sync ext '<SRT_PLAYBACK_URL>'
+```
+
+For more, see [read the docs](/stream/viewing-videos/using-own-player/#play-live-video-in-native-apps-with-less-than-1-second-latency).
```
Comment (dcpena) [inline content/stream/examples/srt_playback.md]:
```suggestion
For more, refer to [Play live video in native apps with less than one second latency](/stream/viewing-videos/using-own-player/#play-live-video-in-native-apps-with-less-than-1-second-latency).
```

File: content/stream/viewing-videos/using-own-player/_index.md [hunk 5]
```diff
@@ -68,6 +68,22 @@ header: Example
...
+
+If you need ultra low latency, and your users view live video in native apps, you can stream live video with [**glass-to-glass latency of less than 1 second**](https://blog.cloudflare.com/magic-hdmi-cable/), by using SRT or RTMPS for playback.
+
+![Diagram showing SRT and RTMPS playback via the Cloudflare Network](/stream/static/stream-rtmps-srt-playback-magic-hdmi-cable.png)
+
+SRT and RTMPS playback is built into [ffmpeg](https://ffmpeg.org/). You will need to integrate ffmpeg with your own video player —  neither [AVPlayer (iOS)](/stream/viewing-videos/using-own-player/ios/) nor [ExoPlayer (Android)](/stream/viewing-videos/using-own-player/android/) natively support SRT or RTMPS playback.
+
+We recommend using [ffmpeg-kit](https://github.com/tanersener/ffmpeg-kit) as a cross-platform wrapper for ffmpeg.
+
+### Examples:
```
Comment (dcpena) [inline content/stream/viewing-videos/using-own-player/_index.md]:
```suggestion
### Examples
```

**Review by Everlag — APPROVED** [review]
Gave it a quick read through, this looks reasonable for me, thanks for taking the time to document this, excited to see how customers use it :D

**Review by dcpena — COMMENTED** [review]
Made a few suggestions but overall 🌟

**Pull request merged**